brevent/src/main/res/values-es/strings.xml

```diff
@@ -193,34 +191,34 @@
     <string name="unsupported_disable_root">Servidor Brevent iniciado correctamente como ROOT.\n\nPor favor habilite \"Permitir ejecutar como ROOT\", en Ajustes/Características experimentales, para iniciar el servidor Brevent en el arranque.</string>
     <string name="unsupported_root">No se puede iniciar ROOT en el servidor Brevent.</string>
     <string name="unsupported_shell">No se puede iniciar servidor Brevent.</string>
     <string name="unsupported_no_local_network">Brevent no puede acceder a la red local.</string>
     <string name="unsupported_dumpsys">No se puede ejecutar DUMPSYS, por favor deshabilite SELinux e inténtelo de nuevo.</string>
     <string name="unsupported_network">No se puede verificar servidor Brevent, por favor deshabilite SELinux e inténtelo de nuevo.</string>
-    <string name="unsupported_exec">Can\'t execute command, please switch to another SU.</string>
-    <string name="unsupported_tasks">Can\'t get recent tasks, please contact ROM provider.</string>
+    <string name="unsupported_exec">No se puede ejecutar comando, por favor cambie a otro SU.</string>
+    <string name="unsupported_tasks">No se pueden obtener tareas recientes, póngase en contacto con el proveedor de su ROM.</string>
 
     <string name="toast_home_tid">Inicio correcto: %d</string>
     <string name="toast_add_package">Brevented \"%1$s\", total %2$d</string>
-    <string name="toast_alipay">Thanks for paying total %1$s RMB via Alipay</string>
-    <string name="toast_alipay2">Restarting Brevent Server, please try later</string>
-    <string name="toast_donate">Thanks for paying %1$s RMB</string>
-    <string name="toast_donate2">Cannot recognize, please try later</string>
+    <string name="toast_alipay">Gracias por pagar, total %1$s RMB via Alipay</string>
+    <string name="toast_alipay2">Reiniciando el servidor de Brevent, inténtelo de nuevo más tarde</string>
+    <string name="toast_donate">Gracias por pagar %1$s RMB</string>
+    <string name="toast_donate2">No se puede reconocer, inténtelo de nuevo más tarde</string>
 
     <string name="logs_description">%1$s\n(Proporcione una descripción de los registros Log.)</string>
 
     <string name="brevent_status_starting">Iniciando servidor Brevent ...</string>
     <string name="brevent_status_not_started">Servidor Brevent no se ha iniciado.</string>
-    <string name="brevent_status_stopped">Brevent server is stopped</string>
-    <string name="brevent_status_unknown">Brevent server maybe stopped</string>
-    <string name="brevent_status_report">Tap here to feedback, and give a detail description</string>
+    <string name="brevent_status_stopped">Servidor Brevent se ha detenido</string>
+    <string name="brevent_status_unknown">Servidor Brevent tal vez detenido</string>
+    <string name="brevent_status_report">Pinche aquí para reportar y dar una descripción detallada</string>
```

Comment:
Yes, it will show when you click the "logs", Brevent will gather the logs and prepare them in the email.

Add "english" or so for "brevent_status_report" and "logs_description". Otherwise, when the email is sent to me, I cannot understand.